sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
 
-3. KEP work in 2021 (1.x, 1.y, 1.z):
+3. KEP work in 2021 (continuous and does not target any release):
 
 <!--
 In future, this will be generated from kubernetes/enhancements kep.yaml files
 1. with SIG as owning-sig or in participating-sigs
 2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
 -->
 
-   - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-   - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-   - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+   - [1326 - Doc policies for third party content](https://git.k8s.io/enhancements/keps/sig-docs/1326-third-party-content-in-docs/README.md)
+
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Require more contributors to form a stable pool for SIG Docs
+   - Require guidelines on how we could ramp up folks nominated to the New Contributor Ambassador role specific to SIG Docs.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+
+   - SIG Docs has a [dashboard](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC) available with site analytics. Some highlights include: 
+       - 2021 Pages views: 111,565,437
+       - Top pages for 2021 (excluding the home page): 
+           - https://kubernetes.io/docs/reference/kubectl/cheatsheet/
+           - https://kubernetes.io/docs/concepts/services-networking/service/
+           - https://kubernetes.io/docs/concepts/services-networking/ingress/
+   - PR velocity and open PR age is tracked in [Devstat](https://k8s.devstats.cncf.io/d/25/open-pr-age-by-repository-group?orgId=1&var-period=q&var-repogroup_name=SIG%20Docs&var-kind_name=All). We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that. For example, based on 12/31/2020 data, we have: Average number of opened PRs 368 with the median opened PRs age 5 days 21 hours 26 minutes 42 seconds
```

Comment:
this is good info: We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that.